config/foreman.migrations/160614145332_foreman_expire_hosts.rb

```diff
@@ -0,0 +1,2 @@
+answers['foreman::plugin::expire_hosts'] ||= false
```

Comment:
%Y's needed, I didn't notice the Kafo recommendation until after I added migrations here.  I used the full year to match Rails migrations and most date formats, and think I still prefer it.